zkvm/src/encoding.rs

```diff
@@ -7,22 +7,6 @@ pub use readerwriter::{ReadError, Reader, WriteError, Writer};
 
 use crate::errors::VMError;
 
```

Comment:
Per suggestion above - let's pub-export Encodable here, so we can continue `use crate::encoding::*`
```suggestion
pub use readerwriter::Encodable;
```

Reply:
Ok.